.pre-commit-config.yaml

```
@@ -42,7 +42,7 @@ repos:
   hooks:
   - id: mypy-local
     name: Run mypy for local Python installation
-    entry: tools/mypy.sh
+    entry: tools/mypy.sh 1 "local"
```

```suggestion
    entry: tools/mypy.sh 0 "local"
```

Should be 0 here to indicate not running in CI

---

The version-locked steps are intended to be run in CI only.